src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -33,6 +37,12 @@ public LogicManager(Model model, Storage storage) {
     public CommandResult execute(String commandText) {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
         Command command = parser.parseCommand(commandText);
+        
+        if(command.modifiesData()){
+        	model.recordStateBeforeChange(command);
+        	model.resetRedoables();
+        }
+        
```

Comment:
In this case, yes, it is due to checkstyle. But you can see why Travis failed by clicking on the "Details" link.

Travis just runs `gradle check`, actually, so you can run that before even submitting the PR to check.